ui/raidboss/data/06-ew/raid/p12s.txt

```diff
@@ -4,3 +4,169 @@ hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+
+# P1 Athena
+0.0 "--sync--" sync /Engage!/ window 0,1
```

Comment:
```suggestion
0.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 0,1
```

I know the Engage is what the make_timeline tool does, but can you switch to "combat start" here.